controllers/subscription/subscription_controller.go

```diff
@@ -115,9 +118,54 @@ func (r *SubscriptionReconciler) Reconcile(ctx context.Context, request ctrl.Req
 		return ctrl.Result{}, nil
 	}
 
+	// Remove code below post 1.38.0 delivery to production
+	// Temporary code to update backend redis 100mln to specific value
+	installation := &integreatlyv1alpha1.RHMI{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "rhoam",
+			Namespace: "redhat-rhoam-operator",
+		},
+	}
+	err := r.Get(context.TODO(), k8sclient.ObjectKey{Name: installation.Name, Namespace: installation.Namespace}, installation)
+	if err != nil {
+		if !k8serr.IsNotFound(err) {
+			return ctrl.Result{}, err
+		}
+	}
+	// Only apply below logic for existing installations, we can't rely on toVersion not present due to possible race condition
+	// between sub and rhmi controller
+	if installation.Status.Version != "" {
```

Comment:
Is it safe to assume that .Status is already in place at this point?

Reply:
It doesn't have to. I believe your concern is about nil pointer but even if status is not set, the code will not return a nil pointer.

Reply:
Ok, thanks. Yes, I was worried about the nil pointer.